opi/pager.py

```diff
@@ -18,23 +18,18 @@ def ensure_scrollarea_bounds(scrollarea_topline_ptr):
 			scrollarea_topline_ptr = min(scrollarea_topline_ptr, max_top_line)
 			return scrollarea_topline_ptr
 		def scrollarea_refresh():
-			scrollarea.refresh(scrollarea_topline_ptr, 0, 0, 0, curses.LINES-3, curses.COLS-1)
+			scrollarea.refresh(scrollarea_topline_ptr, 0, 0, 0, curses.LINES - 3, curses.COLS - 1)
 			# remove artefacts due to smaller status line when scrolling up
 			controlbar.addstr(0, 0, ' ' * curses.COLS)
 			controlbar.addstr(0, 0,
-				"Use arrow keys or PgUp/PgDown to scroll - lines %i-%i/%i %i%%" % (
-					scrollarea_topline_ptr,
-					scrollarea_topline_ptr + (curses.LINES-2),
-					text_len_lines,
-					int(100*scrollarea_topline_ptr / max_top_line)
-				),
+				f'Use arrow keys or PgUp/PgDown to scroll - lines {scrollarea_topline_ptr}-{scrollarea_topline_ptr + (curses.LINES - 2)}/{text_len_lines} {int(100 * scrollarea_topline_ptr / max_top_line)}%',
```

Comment:
Here we get a really long line that is imho harder to read than what we have right now.
So I would prefer to keep it as it is.

Reply:
Changed back